app/models/manageiq/providers/openstack/infra_manager.rb

```diff
@@ -34,6 +34,10 @@ def cloud_tenants
     self.class.none
   end
 
+  def inventory_object_refresh?
+    true
+  end
```

Comment:
OpenStack Infra doesn't have graph refresh yet so we need to make sure that the [`Openstack::InfraManager::Refresher`](https://github.com/ManageIQ/manageiq-providers-openstack/blob/30e0ac3dc8ca4c80bb7ab3fdf7af7323a993a191/app/models/manageiq/providers/openstack/infra_manager/refresher.rb) overrides any methods from the `BaseManager` refresher that call out to `ems.inventory_object_refresh?` aren't being used if we are going to remove that check from core.

Reply:
sorry that was silly